test/common.sh

```diff
@@ -72,20 +76,28 @@ function deploy() {
 		--tenant "${TENANT_ID}" &>/dev/null
 
 	az account set --subscription "${SUBSCRIPTION_ID}"
+}
+
+function deploy_template() {
+	# Check pre-requisites
+	[[ ! -z "${DEPLOYMENT_NAME:-}" ]] || (echo "Must specify DEPLOYMENT_NAME" && exit -1)
+	[[ ! -z "${LOCATION:-}" ]] || (echo "Must specify LOCATION" && exit -1)
+	[[ ! -z "${RESOURCE_GROUP:-}" ]] || (echo "Must specify RESOURCE_GROUP" && exit -1)
+	which kubectl || (echo "kubectl must be on PATH" && exit -1)
+	which az || (echo "az must be on PATH" && exit -1)
 
 	# Deploy the template
-	az group create --name="${INSTANCE_NAME}" --location="${LOCATION}"
+	az group create --name="${RESOURCE_GROUP}" --location="${LOCATION}"
 
 	sleep 3 # TODO: investigate why this is needed (eventual consistency in ARM)
 	az group deployment create \
-		--name "${INSTANCE_NAME}" \
-		--resource-group "${INSTANCE_NAME}" \
+		--name "${DEPLOYMENT_NAME}" \
+		--resource-group "${RESOURCE_GROUP}" \
 		--template-file "${OUTPUT}/azuredeploy.json" \
 		--parameters "@${OUTPUT}/azuredeploy.parameters.json"
-
-	echo "${INSTANCE_NAME} files -> ${OUTPUT}"
 }
 
 function cleanup() {
-	az group delete --no-wait --name="${INSTANCE_NAME}" || true
+	echo "Cleanup"
+	az group delete --no-wait --name="${RESOURCE_GROUP}" --force || true
```

Comment:
Is `--force` actually live yet? Oh, maybe it's in `b3` and I haven't noticed because I'm using a tagged old version? Can you confirm the `--force` works and isn't being silently missed due to the `|| true`?